backend/src/tiedostot/ProjektiTiedostoManager/NahtavillaoloVaiheTiedostoManager.ts

```diff
@@ -11,16 +11,16 @@ import { AsianhallintaSynkronointi } from "@hassu/asianhallinta";
 import { assertIsDefined } from "../../util/assertions";
 
```

Comment:
On kovin tärkeää, että tässä getAineistot-funkkarissa ei kutsuta this.vaihe:tta vaan käytetään tota funktiolle annettua parametria.